src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -1303,7 +1303,7 @@ static void sub_6E1F34(sint16 x, sint16 y, uint16 selected_scenery, sint16* grid
             }
 
             gSceneryPlaceZ = 0;
-            uint16 water_height = tile_element->properties.surface.terrain & TILE_ELEMENT_WATER_HEIGHT_MASK;
+            uint16 water_height = tile_element->properties.surface.terrain & TILE_ELEMENT_SURFACE_WATER_HEIGHT_MASK;
```

Comment:
Could you create a function `surface_element_get_water_height(const rct_tile_element * surfaceElement)` (or a similar name)?